I.pass-tests/04-find-needle/find-needle.js

```diff
@@ -1,3 +1,9 @@
-function findNeedle(words) {}
+function findNeedle(words, word) {
+  if (word === "needle") {
```

Comment:
There is no need to do === "needle" to then use indexOf, you can simply use indexOf.
```suggestion
    return words.indexOf(search);
```